AIX Disk reads is not monotonic

AIX only provides total transfers for disk IO.  We *do* have data on reads/writes in blocks, so OSHI currently uses this to estimate the reads/writes.  From `AIXHWDiskStore`:
```java
// we only have total transfers so estimate read/write ratio from blocks
long blks = stat.rblks + stat.wblks;
this.reads = stat.xfers;
if (blks > 0L) {
    this.writes = stat.xfers * stat.wblks / blks;
    this.reads -= this.writes;
}
this.readBytes = stat.rblks * stat.bsize;
this.writeBytes = stat.wblks * stat.bsize;
```
The problem with this approach is that the integer division for `this.writes` biases writes slightly lower and reads slightly higher, and it is possible for `this.reads` to decrement by 1:
```
Failures (1):
  JUnit Jupiter:DisksTest:testDisks()
    MethodSource [className = 'oshi.hardware.DisksTest', methodName = 'testDisks', methodParameterTypes = '']
    => java.lang.AssertionError: Number of reads from the disk has not been updated
Expected: is a value equal to or greater than <130684L>
     but: <130683L> was less than <130684L>
       org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
       oshi.hardware.DisksTest.testDisks(DisksTest.java:87)
```
The estimation should be updated to cast `blks` to `double` and use `Math.round()` to eliminate the bias.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,7 @@
 
 ##### Bug fixes / Improvements
 * [#1869](https://github.com/oshi/oshi/pull/1869): Ignore mount headers on AIX filesystem - [@dbwiddis](https://github.com/dbwiddis).
+* [#1889](https://github.com/oshi/oshi/pull/1889): Improved calculation of AIX disk reads and writes - [@siddhantdixit](https://github.com/siddhantdixit).
 
 ##### Artifact Removal
 * [#1868](https://github.com/oshi/oshi/pull/1868): Remove shaded jar artifact - [@dbwiddis](https://github.com/dbwiddis).

--- a/oshi-core/src/main/java/oshi/hardware/platform/unix/aix/AixHWDiskStore.java
+++ b/oshi-core/src/main/java/oshi/hardware/platform/unix/aix/AixHWDiskStore.java
@@ -1,7 +1,7 @@
 /*
  * MIT License
  *
- * Copyright (c) 2020-2021 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
+ * Copyright (c) 2020-2022 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
  *
  * Permission is hereby granted, free of charge, to any person obtaining a copy
  * of this software and associated documentation files (the "Software"), to deal
@@ -115,7 +115,7 @@ public final class AixHWDiskStore extends AbstractHWDiskStore {
                 long blks = stat.rblks + stat.wblks;
                 this.reads = stat.xfers;
                 if (blks > 0L) {
-                    this.writes = stat.xfers * stat.wblks / blks;
+                    this.writes = Math.round(stat.xfers * stat.wblks / (double) blks);
                     this.reads -= this.writes;
                 }
                 this.readBytes = stat.rblks * stat.bsize;

